jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -46,7 +46,6 @@ def buildExclusions = [
   [ /vs2017/,                         anyType,     lt(10)  ],
 
   // SmartOS -----------------------------------------------
-  [ /^smartos14/,                     anyType,     gte(8)  ],
```

Comment:
https://ci.nodejs.org/job/node-test-commit-smartos/28512/console started runs on smartos14 for Node.js master since this line was removed but the job wasn't updated to remove the smartos14 labels:
```console
12:28:29 -------------------------------------------------------------------
12:28:29 Node.js major version: 14
12:28:29 Node.js version: 14.0.0
12:28:29 Skipping smartos15-64 for Node.js 14
12:28:29 Skipping smartos16-64 for Node.js 14
12:28:29 Skipping smartos17-64 for Node.js 14
12:28:29 Running nodes
12:28:29 Triggering node-test-commit-smartos » smartos14-64
12:28:29 Triggering node-test-commit-smartos » smartos14-32
12:28:29 Triggering node-test-commit-smartos » smartos18-64
```

All of the `smartos14-*` hosts are currently offline so the build will never complete.

Reply:
@mhdawson has [updated the jobs](https://github.com/nodejs/build/issues/2004).